simde/x86/avx.h

```diff
@@ -502,6 +502,102 @@ simde_mm256_setzero_pd (void) {
   #define _mm256_setzero_pd() simde_mm256_setzero_pd()
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m256
+simde_x_mm256_not_ps(simde__m256 a) {
+    simde__m256_private
+      r_,
+      a_ = simde__m256_to_private(a);
+
+    #if defined(SIMDE_VECTOR_SUBSCRIPT_OPS)
+      r_.i32 = ~a_.i32;
+    #else
+      SIMDE_VECTORIZE
+      for (size_t i = 0 ; i < (sizeof(r_.i32) / sizeof(r_.i32[0])) ; i++) {
+        r_.i32[i] = ~(a_.i32[i]);
+      }
+    #endif
+
+    return simde__m256_from_private(r_);
+}
+
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m256
+simde_x_mm256_select_ps(simde__m256 a, simde__m256 b, simde__m256 mask) {
```

Comment:
Can you please update these to fall back on 128-bit versions if `SIMDE_NATURAL_VECTOR_LE(128)`?

Reply:
Yes, I will update it.